python/src/worker/config/__init__.py

```diff
@@ -106,6 +106,8 @@ def REDIS_CLIENT(self):
             ssl_cert_reqs=None,
         )
 
+if cluster_env == "development" or cluster_env == "test":
+    aws_account_id = "000000000000"
```

Comment:
Can you move the configs from line 133-138 up here too? That means we won't have to directly define the values in `Config` but can just reassign values for the variables.